x-element.js

```diff
@@ -1010,7 +1010,7 @@ export default class XElement extends HTMLElement {
 
 /** Internal implementation details for template engine. */
 class TemplateEngine {
-  static #UNSET = Symbol('unset'); // Ensures a unique, initial comparison.
```

Comment:
I decided that the labels for the Symbols isn’t actually additive. I could potentially be convinced otherwise, but the code context is always sufficient to understand the meaning.